girder/api/v1/user.py

```diff
@@ -196,6 +197,15 @@ def deleteUser(self, user):
         self.model('user').remove(user)
         return {'message': 'Deleted user %s.' % user['login']}
 
+    @access.admin
+    @autoDescribeRoute(
+        Description('Get the number of users.')
+    )
+    def getUserNumber(self):
```

Comment:
This method name is a bit unclear, when I read it, I interpret it to be get a number from a specific user (which doesn't exist, but in many data models this would exist).

Please change it to `getUserCount`